src/app/Fake.DotNet.Fsdocs/Fsdocs.fs

```diff
@@ -109,14 +117,18 @@ module Fsdocs =
 
             /// Port to serve content for <c>http://localhost</c> serving.
             Port: int option
+
+            /// Build Commands
+            BuildCommandParams : BuildCommandParams option
         }
 
         /// Parameter default values.
         static member Default =
             { NoServer = None
               NoLaunch = None
               Open = None
-              Port = None }
+              Port = None
+              BuildCommandParams = None }
```

Comment:
[WatchCommand inherits CoreBuildOptions](https://github.com/fsprojects/FSharp.Formatting/blob/e665aa95ab897e5e2c543e5847f3820b3e215e92/src/fsdocs-tool/BuildCommand.fs#L1862). Simplest addition without copying this.